pytorch_lightning/accelerators/cpu.py

```diff
@@ -39,6 +40,19 @@ def get_device_stats(self, device: _DEVICE) -> dict[str, Any]:
         """CPU device stats aren't supported yet."""
         return {}
 
+    @staticmethod
+    def parse_devices(devices: int | str | list[int]) -> int | str | list[int]:
+        """Accelerator Parsing logic."""
+        return devices
+
+    @staticmethod
+    def get_parallel_devices(devices: int | str | list[int]) -> list[torch.device]:
+        """Gets parallel devices for the given Accelerator."""
+        if isinstance(devices, int):
+            return [torch.device("cpu")] * devices
```

Comment:
> [torch.device("cpu")] * devices

not sure if there will be any side-effect, but this creates the references N times rather than independent instances.

Reply:
Not introducing any new changes in this PR, just moving the existing logic to accelerators.

Reply:
but I think you can change it here since it's not a big one... or in a follow-up if you prefer

Reply:
Follow up!